onnx/inliner/inliner.h

```diff
@@ -79,5 +79,64 @@ void InlineSelectedFunctions(ModelProto& model, const FunctionIdSet& to_inline);
 // functions that use opset versions that are compatible with the model.
 void InlineLocalFunctions(ModelProto& model, bool convert_version = false);
 
+/**
+ * @brief A utility class for renaming variables during graph inlining operations.
+ *
+ * This class provides a simplified interface to the InliningRenamer functionality,
+ * allowing users to bind formal parameter names to actual parameter names and
+ * rename nodes with unique prefixes.
+ */
+class Renamer {
+ public:
+  /**
+   * @brief Constructs a Renamer with the given prefix.
+   *
+   * @param prefix The prefix to add to intermediate variable names to ensure uniqueness.
+   * @param graph The graph context for name generation.
+   */
+  explicit Renamer(const std::string& prefix, const GraphProto& graph);
+
+  /**
+   * @brief Constructs a Renamer with the given prefix.
+   *
+   * @param prefix The prefix to add to intermediate variable names to ensure uniqueness.
+   * @param function The function context for name generation.
+   */
+  explicit Renamer(const std::string& prefix, const FunctionProto& function);
+
+  /**
+   * @brief Destructor.
+   */
+  ~Renamer();
+
+  /**
+   * @brief Binds a formal parameter name to an actual parameter name.
+   *
+   * @param formal_name The formal parameter name in the source graph.
+   * @param actual_name The actual parameter name to use in the target.
+   */
+  void BindName(const std::string& formal_name, const std::string& actual_name);
+
+  /**
+   * @brief Renames all variables in the given node according to the current bindings.
+   *
+   * @param node The node to rename. Input and output names will be updated in place.
+   */
+  void RenameNode(NodeProto& node);
+
+  /**
+   * @brief Creates a unique name based on the given suggested name.
+   *
+   * @param suggested_name The suggested name to make unique.
+   * @return A unique name based on the suggested name.
+   */
+  std::string CreateUniqueName(const std::string& suggested_name);
```

Comment:
I don't think we need to expose the method to create unique names. We can drop it.